### ISSUE TITLE
How to train from stratch and evaluate on modelnet40 for object classification?

### ISSUE THREAD

**Issue by TinaJin0228** [Pang-Yatian/Point-MAE#23]
Thanks for your nice work and detailed open source code, they helped a lot.
My question is, I want to train from stratch on the ModelNet40 for object classification and reproduce the result, but I don't find how to do it.
I noticed that the result is quoted from point-BERT's article, so maybe I should trun to point-BERT‘s work for this?
***
<img width="500" alt="image" src="https://user-images.githubusercontent.com/60577222/181729134-17e0c9f0-5744-40a9-a03e-d5105ac3cdcd.png">
___
<img width="500" alt="image" src="https://user-images.githubusercontent.com/60577222/181729211-fa8b11d4-3093-4772-b1d2-3530033d647a.png">

**Comment by Pang-Yatian** [comment]
Hi,

You can simply run finetuning command without passing in --ckpts.

**Comment by TinaJin0228** [comment]
> Hi,
> 
> You can simply run finetuning command without passing in --ckpts. We directly use the result from Point-BERT's paper.

thank you

**Comment by TinaJin0228** [comment]
> Hi,
> 
> You can simply run finetuning command without passing in --ckpts. We directly use the result from Point-BERT's paper.

I ran the cammand as you mentioned and got a result of accuracy of 93.3144%
Besides, the result I got with pretraining is 93.19%
Do you have any idea of why this happend? I am really confused

**Comment by Pang-Yatian** [comment]
I have no idea.

**Comment by kabouzeid** [comment]
The model used in Point-BERT is not exactly the same as in Point-MAE. Point-MAE uses a heavier MLP for the classifier part for example. Indeed it seems like for ModelNet40, MAE and other self supervised pretraining methods I've experimented with perform *worse* than training from scratch. I can't explain why. This is not the case for ScanObjectNN for example where MAE does work very well!

**Comment by AnukritiSinghh** [comment]
Hi, I want to test on modelnet40, can you share the exact command for that?

**Comment by TinaJin0228** [comment]
> Hi, I want to test on modelnet40, can you share the exact command for that?

i just ran the test vote command as the author offered:
```
CUDA_VISIBLE_DEVICES=<GPUs> python main.py --test --config cfgs/finetune_modelnet.yaml \
--exp_name <output_file_name> --ckpts <path/to/best/fine-tuned/model>
```
